packages/runtime/container-runtime/src/opLifecycle/remoteMessageProcessor.ts

```diff
@@ -113,8 +114,21 @@ export class RemoteMessageProcessor {
 				this.processorBatch.length === 0,
 				0x9d4 /* Processor batch should be empty on grouped batch */,
 			);
+			const groupedMessages = this.opGroupingManager.ungroupOp(message).map(unpack);
+			// If the batch is empty, we need to return the sequence number aside
+			if (groupedMessages.length === 0) {
+				assert(
+					message.sequenceNumber !== undefined,
+					"Empty grouped batch has no sequence number",
```

Comment:
This is ok, but it's technically overkill - the type of `sequenceNumber` is `number`, non-optional, so it'll be there.  And if not, this isn't necessarily the place we need to catch it.